src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTabOrFirstId = tabs.find(tab => tab.id === activeTabId)
+    ? activeTabId
+    : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                data-cy="Tab"
+                className={`${activeTabOrFirstId === tab.id ? 'is-active' : ''}`}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (activeTabOrFirstId !== tab.id) {
+                      onTabSelected(tab.id);
+                    }
+                  }}
+                >
+                  {tab.title}
+                </a>
+              </li>
```

Comment:
Each `<li>` element rendered in the `map` function should have a unique `key` prop. This helps React optimize rendering and update processes. You can use `tab.id` as the `key` since it is unique for each tab.